### PR TITLE
Pr helm for service monitor

### PR DESCRIPTION
**Pull request by rinckm**
Add service monitors to steward helm chart

### CODE REVIEW

File: charts/steward/templates/service-monitor-run.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.metrics.serviceMonitors.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: steward-run-controller-metrics 
+  namespace: steward-system
```
Comment (alxsap) [inline charts/steward/templates/service-monitor-run.yaml]: Namespace is configurable! see `targetNamespace.name`

File: charts/steward/templates/service-monitor-tenant.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.metrics.serviceMonitors.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: steward-tenant-controller-metrics 
+  namespace: steward-system
```
Comment (alxsap) [inline charts/steward/templates/service-monitor-tenant.yaml]: Namespace is configurable! see `targetNamespace.name`

File: charts/steward/values.yaml [hunk 3]
```diff
@@ -25,7 +25,9 @@ runController:
   nodeSelector: {}
   affinity: {}
   tolerations: []
-
+  monitoring:
+    enabled: true
```
Comment (alxsap) [inline charts/steward/values.yaml]: Looks like it is not used

File: charts/steward/values.yaml [hunk 4]
```diff
@@ -49,6 +51,15 @@ tenantController:
   nodeSelector: {}
   affinity: {}
   tolerations: []
+  monitoring:
+    enabled: true
```
Comment (alxsap) [inline charts/steward/values.yaml]: Looks like it is not used

File: charts/steward/templates/service-monitor-run.yaml (hunk 1 above)
Comment (anbrsap) [reply charts/steward/templates/service-monitor-run.yaml]: Fixed with 78dadc3.

File: charts/steward/values.yaml (hunk 3 above)
Comment (anbrsap) [reply charts/steward/values.yaml]: Removed with 78dadc3.

File: charts/steward/values.yaml (hunk 4 above)
Comment (anbrsap) [reply charts/steward/values.yaml]: Removed with 78dadc3.

**Pull request merged**